app/cdap/components/GridTable/components/NestedMenu/types.ts

```diff
@@ -25,3 +25,17 @@ export interface INestedMenuProps {
   open?: boolean;
   handleMenuOpenClose?: (title?: string) => void;
 }
+
+interface IOptions {
+  value: string;
+  label: string;
+  options?: IOptions[];
+  supported_dataType: string[];
```

Comment:
this should be camelCase
i remember us discussing that it is used like this by the old code
if so, please ignore otherwise we need to fix this